apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -454,7 +459,7 @@ private String partDFieldFor(CcwCodebookVariable month) {
     if (hasHICN(identifiers)) joinsClause += "left join fetch b.beneficiaryHistories ";
 
     String query =
-        "select b from Beneficiary b "
+        "select distinct b from Beneficiary b "
```

Comment:
This same change would also be needed in the V2 resource provider.

Reply:
fix that